config_sample.yml

```diff
@@ -10,7 +10,9 @@ captchaPath: c:/selenium-config/captcha.txt #Path to your captcha.txt file
 qs: null
 
 #Config related with mobile devices
-platfrom: null #acceptable values: IOS and ANDROID
+platfrom: ANDROID #acceptable values: IOS and ANDROID
```

Comment:
do you want Android as default ? You we might want null as it is in other configurable fields
I have the same concerns for appium-ip and deviceName fields